library/lib/build.js

```diff
@@ -194,6 +197,16 @@ module.exports = function build({ watch }) {
         ]}]
       },
       plugins: [
+        ...watch ? [] : [
+          new webpack.LoaderOptionsPlugin({
```

Comment:
Is this only needed to support the Uglify plugin? If that's the case I think I agree with you on having them as a pair of plugins. To make it more obvious you could pull them to a function, or maybe even better: wrap them in a plugin.

(from the top of my head)
```
const plugin = {
  apply: compiler => {
    pluginA.apply(compiler)
    pluginB.apply(compiler)
  }
}
```